apps/src/fish/Fish.js

```diff
@@ -129,6 +130,12 @@ Fish.prototype.initMLActivities = function () {
 
   const {initAll} = require('@code-dot-org/ml-activities');
 
+  // Localize
+  const msg = {};
+  for (const [key, msgFunction] of Object.entries(fishMsg)) {
+    msg[key] = (...args) => localization.translate(msgFunction(...args));
+  }
```

Comment:
nit to use Object.reduce instead of mutating msg, but either or works for me.

```suggestion
  // Localize
const msg = Object.entries(fishMsg).reduce(
  (acc, [key, msgFunction]) => {
    acc[key] = (...args) => localization.translate(msgFunction(...args));
    return acc;
  },
  {}
);
```

Reply:
I never think of side-effect reduces like this.